Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingAlgorithm.hpp

```diff
@@ -64,6 +64,10 @@ class SeedingAlgorithm final : public IAlgorithm {
     // number of phiBin neighbors at each side of the current bin that will be
     // used to search for SPs
     int numPhiNeighbors = 1;
+
+    // Connect custom selections on the space points or to the doublet
+    // compatibility
```

Comment:
```suggestion
    /// Connect custom selections on the space points or to the doublet
    /// compatibility
```

same above but not modified by you